_rules/aaa1bf.md

```diff
@@ -0,0 +1,98 @@
+---
+id: aaa1bf
+name: auto-play audio does not exceed 3 seconds
+rule_type: atomic
+description: |
+  auto-play audio content does not output audio for more than 3 seconds
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Audio output
+  - Visual output
+authors:
+  - Anne Thyme Nørregaard
+  - Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>`, `<video>` or `<source>` elements, with a `src` attribute referencing content with a duration of more than 3 seconds, that has an `autoplay` attribute equal to `true`, and that has both `paused` and `muted` attributes equal to `false`.
+
+## Expectation
+
+For each test target the total audio output does not last more than 3 seconds.
+ 
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
```

Comment:
Don't you want to include at least the part about the elements not being properly announced? (I agree that the keyboard control support is not relevant for this rule).

Reply:
I think that AT inconsistently announcing controls is not what is being tested here so I would prefer to keep the note on the composite and controls atomic rules that do test for controls

Reply:
That's fair